js/app.js

```diff
@@ -0,0 +1,17 @@
+/*global window, console, document, Image
+*/
+$(function () {
+  'use strict';
+
+  // unselects the instructions button after modal is hidden
+  // 10ms delay before blur is triggered
+  $('#myModal').on('hidden.bs.modal', function() {
+    setTimeout(function () {
+      $('#instructions').trigger('blur');
+    }, 10);
+  })
+
+  // initialize game
+  var game = createGame();
+  game.init();
```

Comment:
food for thought: if every time you create a game, you have to initialize it, shouldn't the code of init be either implemented or called from the constructor itself?